rtCommon/bidsCommon.py

```diff
@@ -335,11 +335,16 @@ def niftiHeadersAppendCompatible(header1: dict, header2: dict):
             return (False, errorMsg)
```

Comment:
This file won't have any changes in the final PR, as I'll rebase it off master after merging in #33